Maestro/src/Microsoft.DotNet.Maestro/Config.cs

```diff
@@ -29,14 +32,39 @@ public string WebhookSecretToken
             get { return _webhookSecretToken.Value; }
         }
 
-        public string VSoUser
+        public string VstsUsername
         {
-            get { return _vsoUser.Value; }
+            get { return _vstsInstanceUserName.Value; }
         }
 
-        public string VSoPassword
+        public string GetPassword(string instance)
         {
-            get { return _vsoPassword.Value; }
+            if (_vstsCreds.ContainsKey(instance))
+            {
+                return _vstsCreds[instance];
```

Comment:
Is there any chance of multiple threads calling this?

If not, then I'd be a tiny bit happier with `TryGetValue` as a matter of style, although I'm sure perf doesn't matter here.

Reply:
@dagood I think you could get multiple threads if multiple handlers were called at once (actually, this should probably be a thread safe dictionary)

Reply:
I'm a big fan of `ConcurrentDictionary`'s API, fwiw.